boards/arm/rpi_pico/doc/index.rst

```diff
@@ -85,13 +94,22 @@ hardware features:
      - :kconfig:option:`CONFIG_SERIAL`
      - :dtcompatible:`raspberrypi,pico-uart-pio`
 
+The rpi_pico_w board configuration will additionally support WiFi and
+Bluetooth hardware features (when drivers become available):
+
 Pin Mapping
 ===========
 
 The peripherals of the RP2040 SoC can be routed to various pins on the board.
 The configuration of these routes can be modified through DTS. Please refer to
 the datasheet to see the possible routings for each peripheral.
 
+External pin mapping on the Pico W is identical to the Pico, but note that internal
+RP2040 GPIO lines 23, 24, 25, and 29 are routed to the Infineon module on the W.
+Since GPIO 25 is rounted to the on-board LED on the Pico, but to the Infineon module
```

Comment:
"rounted" is a typo

Reply:
Thanks. All 3 changes made as requested and pushed.